include/deal.II/lac/la_parallel_block_vector.h

```diff
@@ -172,7 +172,7 @@ namespace LinearAlgebra
        * Copy operator: fill all components of the vector with the given
        * scalar value.
        */
-      BlockVector &operator = (const value_type s);
+      BlockVector &operator = (const value_type s) override;
```

Comment:
are this and the next function `virtual`? If so, would you mind making this explicit as well?